src/core/edm/reducers/getOrgDataSetSizeReducer.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * @flow
+ */
+
+import { Map } from 'immutable';
+import { RequestStates } from 'redux-reqseq';
+import type { SequenceAction } from 'redux-reqseq';
+
+import { ENTITY_SET_SIZE_MAP, REQUEST_STATE } from '../../redux/constants';
+import { GET_ORG_DATA_SET_SIZE, getOrgDataSetSize } from '../actions';
+
+export default function reducer(state :Map, action :SequenceAction) {
+
+  return getOrgDataSetSize.reducer(state, action, {
+    REQUEST: () => state
+      .setIn([GET_ORG_DATA_SET_SIZE, REQUEST_STATE], RequestStates.PENDING)
+      .setIn([GET_ORG_DATA_SET_SIZE, action.id], action),
+    SUCCESS: () => {
+      if (state.hasIn([GET_ORG_DATA_SET_SIZE, action.id])) {
+        const {
+          dataSetId,
+          organizationId,
+          size
+        } = action.value;
```

Comment:
No change require, just a thought. You have access to `dataSetId` and `organizationId` in the stored action from `.request()` so you don't need to pass these again in `.success()`, but I understand the convenience.